pyfar/plot/_line.py

```diff
@@ -110,6 +110,19 @@ def _set_axlim(ax, setter, low, high, limits):
         setter((min(limits[0], low), max(limits[1], high)))
 
 
+def _lower_frequency_limit(signal):
+    """Return the lower frequency limit for plotting."""
+    if isinstance(signal, Signal):
+        lower_frequency_limit = max(20, signal.frequencies[1])
```

Comment:
Why is there a difference for FrequencyData and Signals here?

Reply:
This is a design choice. The idea is this:
- We do not want to plot 0 Hz, because we most often plot on a log freq scale that does not have 0 Hz
- We want to plot starting at 20 Hz, if we have data down to this
- If we do not have data down to 20 Hz, we plot the lowest available frequency.

For signals, `frequencies[0]=0` in all cases. If we used that for checking, `_lower_frequency_limit()` will always return 20 Hz for signals, although the lowest available non-zero frequency might be higher than that.